src/components/cursor.js

```diff
@@ -1,94 +1,162 @@
 var registerComponent = require('../core/component').registerComponent;
+var THREE = require('../lib/three');
 var utils = require('../utils/');
 
+var EVENTS = {
+  CLICK: 'cursor-click',
```

Comment:
Possibly. It may be unexpected for component developers to emit an event, try to add a listener, and not knowing that it's prefixing under-the-hood.